### PR TITLE
Add .gitignore, LICENSE and README.md

### LINKED ISSUE #2: Add .gitignore, README.md and License to the repository
This repository needs:
- `.gitignore` to prevent accidental addition of binary files and other undesired artifacts
- `README.md` to describe this repository and its contents
- `LICENSE` to conform with other NOAA-EMC and ufs-community repository requirements.

### PR DESCRIPTION
**Pull request by aerorahul**
Closes #2

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,504 @@
+                  GNU LESSER GENERAL PUBLIC LICENSE
```
Comment (RussTreadon-NOAA) [inline LICENSE]: I have no idea which GNU license to use.  I see a variety of licenses in NOAA-EMC repositories.   Should we have consistency across NOAA-EMC repositories?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# GSI-fix
+Static ASCII files used in the [GSI](https://github.com/NOAA-EMC/GSI) by various applications to describe the characteristics of the system.
```
Comment (RussTreadon-NOAA) [inline README.md]: The ASCII files are actually less static than the binary GSI fix files.   Changes to binary GSI fix files are less common.  Changes to ASCII GSI fix files are more common given the uses for these fix files as stated on lines 4 to 9.   We created GSI-fix to track changes in ASCII GSI-fix files.

Suggestion:  remove `Static` and simply start with `ASCII files ...`

**Review by RussTreadon-NOAA — COMMENTED** [review]
Thank you @aerorahul for opening this PR.   I confirmed that the list of binary fix files in `.gitignore` is correct.   One question for `LICENSE` and one minor comment for `README.md`.

File: README.md (hunk 2 above)
Comment (aerorahul) [reply README.md]: Good point.  I incorporated this suggestion.

File: LICENSE (hunk 1 above)
Comment (aerorahul) [reply LICENSE]: LGPLv3 is the one recommeneded and used. If others are using something else, they should be updated.
I have updated it here.

**Review by RussTreadon-NOAA — APPROVED** [review]
Thank you for your responses.   Changes look good to me.

**Comment by RussTreadon-NOAA** [comment]
@aerorahul , would you like me to merge this PR?  I'm more than willing ... just cross checking with you.

**Comment by aerorahul** [comment]
> @aerorahul , would you like me to merge this PR? I'm more than willing ... just cross checking with you.

That would be great.  Thank you.

**Pull request merged**